lib/project_types/script/layers/infrastructure/script_service.rb

```diff
@@ -26,7 +26,7 @@ def push(
             extensionPointName: extension_point_type.upcase,
             title: script_name,
             description: description,
-            configUi: config_ui.content,
+            configUi: config_ui&.content,
```

Comment:
`config_ui` is `nil` when no `config_ui_file` entry exists in the `.shopify-cli.yml`, right?

Reply:
yes exactly!